release-notes/elastic-security/known-issues.md

```diff
@@ -17,6 +17,18 @@ Known issues are significant defects or limitations that may impact your impleme
 
 :::
 
+:::{dropdown} Installing an {{elastic-defend}} integration or a new agent policy upgrades installed prebuilt rules, reverting user customizations and overwriting user-added actions and exceptions
```

Comment:
```suggestion
:::{dropdown} Installing an {{elastic-defend}} integration or a new agent policy upgrades installed prebuilt rules, reverting user customizations and overwriting user-added actions and exceptions.
```

Reply:
Chatted with Ben; disregarding this one to be compliant with other dropdown titles.